apps.js

```diff
@@ -41,63 +32,22 @@ const external = createApp('external', app => {
   app.use('/manage', routes.manage);
 });
 
+const report = FabrikApp.createApp('report', app => {
```

Comment:
is this required? where is `app.js` used? if this is not used wouldnt it be better to get rid of this file?

Reply:
It is used in unit tests, what we can do is to move it to test directory.